x-pack/plugins/security_solution/server/lib/detection_engine/rule_management/logic/import/import_rules_utils.ts

```diff
@@ -141,6 +141,7 @@ export const importRules = async ({
                       exceptions_list: [...exceptions],
                     },
                     allowMissingConnectorSecrets,
+                    shouldIncrementRevision: false,
```

Comment:
Why exactly it is false here?
When testing import I noticed a bug and I suspect this might be causing it